platforms/jvm/plugins-jvm-test-suite/src/main/java/org/gradle/api/testing/toolchains/internal/JUnitJupiterTestToolchain.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.testing.toolchains.internal;
+
+import org.gradle.api.artifacts.Dependency;
+
+import java.util.Collections;
+
+/**
+ * A {@link JUnitPlatformTestToolchain} that uses the JUnit Jupiter test engine.
+ *
+ * @since 8.5
+ */
+abstract public class JUnitJupiterTestToolchain extends JUnitPlatformTestToolchain<JUnitJupiterToolchainParameters> {
+    public static final String DEFAULT_VERSION = "5.8.2";
```

Comment:
Should probably add javadoc to this public field of a (eventually) public class. Applies to other toolchain classes as well

Reply:
Yeah, I think that makes sense.  Done.